### PR TITLE
Color palette

### PR DESCRIPTION
**Pull request by Horay**
When merged this PR will add a color palette

### CODE REVIEW

File: src/theme/index.tsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
 import { createBreakpoint, createMap } from "styled-components-breakpoint";
+import { palette } from "./palette";
 
 type Colors = {
   darks: Array<string>;
   lights: Array<string>;
   [color: string]: string | any;
 };
 
+type Color = {
```
Comment (nowyDEV) [inline src/theme/index.tsx:10]:
```suggestion
type Colors = {
```

File: src/theme/index.tsx [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+  background: { [key: string]: string };
+  text: { [key: string]: string };
+  border: { [key: string]: string };
+  interactive: { [key: string]: string };
+  support: { [key: string]: string };
+  [key: string]: string | any;
```
Comment (nowyDEV) [inline src/theme/index.tsx:16]: every property should be listed, don't use indexed types

File: src/theme/palette.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
+export const palette: any = {
```
Comment (nowyDEV) [inline src/theme/palette.tsx:1]: Don't use any, everything should be typed

File: src/theme/palette.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    orange100: "#ffe0b2",
+    orange200: "#ffcc80",
+    orange300: "#ffb74d",
+    orange400: "#ffa726",
+    orange500: "#ff9800",
+    orange600: "#fb8c00",
+    orange700: "#f57c00",
+    orange800: "#ef6c00",
+  },
+};
```
Comment (nowyDEV) [inline src/theme/palette.tsx:144]:
```suggestion
} as const;
```

**Review by janosio-nordcloud — APPROVED** [review]

File: src/theme/index.tsx [hunk 5]
```diff
@@ -1,10 +1,86 @@
 import { createBreakpoint, createMap } from "styled-components-breakpoint";
+import { palette } from "./palette";
 
 type Colors = {
   darks: Array<string>;
   lights: Array<string>;
   [color: string]: string | any;
 };
+type backgroundColor = {
```
Comment (nowyDEV) [inline src/theme/index.tsx:9]: types names should be capitalized

File: src/theme/index.tsx [hunk 6]
```diff
@@ -1,10 +1,86 @@
...
+  ui01: string;
+  ui02: string;
+  ui03: string;
+  ui04: string;
+  ui05: string;
+  error: string;
+  warning: string;
+  success: string;
+  info: string;
+};
```
Comment (nowyDEV) [inline src/theme/index.tsx:20]: Add empty line between type declarations

**Review by nowyDEV — APPROVED** [review]

**Review by janosio-nordcloud — APPROVED** [review]

**Pull request merged**